securedrop/journalist_app/admin.py

```diff
@@ -33,9 +35,30 @@ def manage_config():
             f = form.logo.data
             static_filepath = os.path.join(config.SECUREDROP_ROOT,
                                            "static/i/logo.png")
+            temp_static_filepath = os.path.join(config.SECUREDROP_ROOT,
+                                                "static/i/temp_logo.png")
```

Comment:
I'm not sure we should save temp files in `./static`. I think we can use the `SECUREDROP_DATA_ROOT` and make sure there's a `tmp` in there. This would provide us with safe place for temp files.